cluster/gce/util.sh

```diff
@@ -2061,6 +2067,49 @@ function create-etcd-certs {
   popd
 }
 
+# Generates SSL certificates for etcd-client and kube-apiserver communication. Uses cfssl program.
+#
+# Assumed vars:
+#   KUBE_TEMP: temporary directory
+#   NUM_NODES: number of nodes in the cluster
```

Comment:
This doesn't seem to be used anywhere

Reply:
Thanks! I just removed that. Lesson learned, I should always check and make sure the comments are valid, especially copy comments from other functions...